src/plugins/discover/public/application/context/services/context_state.ts

```diff
@@ -282,3 +313,21 @@ function createInitialAppState(
     uiSettings
   );
 }
+
+/**
+ * Helper function to return the initial global state, which is a merged object of url state and
+ * default state
+ */
+function createInitialGloblalState(urlState: GlobalState): GlobalState {
```

Comment:
```suggestion
function createInitialGlobalState(urlState: GlobalState): GlobalState {
```

Reply:
Will be fixed in next commit, no more 'globlal' 😁